store/index.js

```diff
@@ -109,11 +108,6 @@ export default function configureStore(initialState) {
                             // used by the application such as extra and redirect_to.
                             window.location.href = `${basePath}${window.location.search}`;
 
-                            store.dispatch({
```

Comment:
This is the other important part. This is the action that's dispatched to cause the state tree to be overwritten